frontend/src/components/CircularMenu/CircularMenu.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from "react";
+import "./CircularMenu.scss";
+import PropTypes from "prop-types";
+import { isIOS } from "react-device-detect";
+
+const CircularMenu = ({
+  itemsAmount,
+  x,
+  y,
+  row,
+  column,
+  updateBoard,
+  hideMenu
+}) => {
+  const tilesArray = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const [isOpen, setIsOpen] = useState(false);
+
+  const displayMenu = () => {
+    if (isOpen) {
+      hideMenu();
+      setIsOpen(false);
+    } else {
+      setIsOpen(true);
+    }
+  };
+
+  useEffect(() => {
+    if (isIOS) {
+      displayMenu();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Remove this line and instead of using `displayMenu()` use `setIsOpen` directly (you can safely add this method to the deps).